crates/democracy/src/tests/public_proposals.rs

```diff
@@ -28,6 +28,24 @@ fn deposit_for_proposals_should_be_taken() {
     });
 }
 
+#[test]
```

Comment:
Can you add a test for cancelling as root as well? Not that I'm worrying about correctness, but just for completeness